core/src/test/scala/org/apache/spark/deploy/master/ui/MasterWebUISuite.scala

```diff
@@ -35,6 +35,7 @@ import org.apache.spark.rpc.{RpcEndpointRef, RpcEnv}
 import org.apache.spark.util.Utils
 
 class MasterWebUISuite extends SparkFunSuite {
+  import MasterWebUISuite._
```

Comment:
Do you include new stuffs? I didn't see any one.

Reply:
Yes, `MasterWebUISuite.sendHttpRequest `is imported here like `master` branch.